### ISSUE TITLE
BUG: MongoPipelineStatus.isPrepared does not guarantee prepare status

### ISSUE THREAD

**Issue by ebbesson** [Findwise/Hydra#167]
How to reproduce:
Initiate Hydra with a clean database. Add stages and configurations
Start hydra
Hydra creates database and 4 collections (documents,oldDocuments,hydra,stages)
prepared is set to true. 
Shutdown Hydra core
Now drop oldDocuments in mongo. - prepared state is now compromised

Start hydra again and hydra does not re-create the oldDocuments on connect, due to prepare status being set to true.

**Comment by rgraneru** [comment]
Yes, I've also noticed this. But we won't fix it since we shouldn't really
hack in the db like that. I do all the time, but in prod it shouldn't be a
problem.
Den 28. feb. 2013 10:56 skrev "Magnus Ebbesson" notifications@github.com
følgende:
